dts/bindings/mtd/jedec,spi-nor.yaml

```diff
@@ -19,11 +19,71 @@ properties:
   has-be32k:
     type: boolean
     required: false
-    description: Indicates the device supports the BE32K command
+    description: Indicates the device supports the BE32K (0xD8) command
 
-  size:
+  requires-ulbpr:
+    type: boolean
+    required: false
+    description: Indicates the device requires the ULBPR (0x98) command
```

Comment:
Could maybe use one of the multi-line syntaxes in https://yaml-multiline.info/ (just discovered that site... pretty specialized) and put a period at the end of the first lines.

Reply:
Tell me if the output for these long descriptions gets ugly btw. Could improve it in that case.

Reply:
I'd looked at that site, but it didn't seem to add anything beyond what's normally used.  I think I've used `>` in previous bindings; `>-` doesn't add anything unless the value is actually used in generated documentation somewhere.  Is it?

The vast majority of description text in `dts/bindings` does not have a terminating period.

Reply:
Could skip the period if it's meant to be some kind of title though I guess...

Reply:
> I'd looked at that site, but it didn't seem to add anything beyond what's normally used.

Nice to have the first line line up with the other lines I think.

> I think I've used `>` in previous bindings; `>-` doesn't add anything unless the value is actually used in generated documentation somewhere. Is it?

Should end up in the generarated header/configuration file. Think it's just `rstrip()`ed at the moment.

Reply:
Without looking, I suspect that it gets a bit ugly.

I'll make a PR tomorrow so that it gets output as

```c
/*
 * Long description
 * yada yada
 * yada
 */
```

for multi-line descriptions and as

```c
/* yada yada */
```

for single-line descriptions.

Reply:
I think all these yaml issues have been addressed.